tst/testspecial/func-overflow.g

```diff
@@ -0,0 +1,3 @@
+strlist := String(List([1..2097152], x -> x));;
+func := Concatenation("lf := function() return ", strlist, "; end;");;
```

Comment:
Why is this in testspecial and not part of the `.tst` file above?
```
gap> strlist := String(List([1..2097152], x -> x));;
gap> func := Concatenation("lf := function() return ", strlist, "; end;");;
gap> Read(InputTextString(func));;
Error, function too large for parser
```
should work fine there, no?

Reply:
The reason this was here is previously when this PRed "panic"ed, I couldn't do that in a normal test. Now it's an error, I can (I also added a record test as well. I don't know if anyone would ever create such a huge record in code, but it doesn't hurt to catch it!)